src/kbmod/analysis_utils.py

```diff
@@ -54,8 +55,10 @@ def load_images(
         -------
             stack : `kbmod.ImageStack`
                 The stack of images loaded.
-            img_info : `ImageInfo`
-                The information for the images loaded.
+            wcs_list : `astropy.wcs.WCS`
```

Comment:
This is a great question I've been struggling with too. WCS is un-plural-izable, but I really dislike the Hungarian notation. No change requested here, but  as a point for the future, I am, however, appinionating that we can just use `wcs` for both and usually the context makes sense of it. There are no image collections or stack searches that do not hold multiple `wcs` and no raw images that can hold more than one `wcs`. I don't know if you've seen examples where we couldn't default to just `wcs` for both cases?

Since this is going to be refactored soon-ish again to put in image/metadata collection, it's fine as it is.